lib/puppetdb_foreman.rb

```diff
@@ -1,3 +1,4 @@
+require 'puppetdb_foreman/engine'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.